src/backend_model.h

```diff
@@ -218,6 +235,12 @@ class TritonModel : public Model {
   // persist as along as this object is retained by this model.
   std::shared_ptr<LocalizedPath> localized_model_dir_;
 
+  // Python-based backends provide model.py in their directory.
+  // This `model.py` is later reused by models,
+  // using this python-based backend.
+  // We provide the location of common model.py, when model is initialized.
+  std::string py_backend_model_path_ = "";
```

Comment:
Could this be added to the `TritonBackend` class instead?

Reply:
this [comment](https://github.com/triton-inference-server/core/pull/269#discussion_r1341875824)